### PR TITLE
Prepare CRAN release v2.1.0

### PR DESCRIPTION
**Pull request by fweber144**
Here are my final changes for the CRAN release of version 2.1.0. As soon as @paul-buerkner has approved this (you can also merge it if you want), I'll merge the resulting `develop` into `master`, submit to CRAN, and (if approved by CRAN) update `gh-pages`.

### CODE REVIEW

File: cran-comments.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+## Maintainer change
```
Comment (paul-buerkner) [inline cran-comments.md:1]: In brms I have cran-comments.md under .gitignore. I remember Hadely Wickham recommending to not commit it but it is some time ago that I have read it. Could you check how tidyverse packages to it?

**Review by paul-buerkner — COMMENTED** [review]
Looks good. I have only one question about comming cran-comments.md. After this has been resolve, I will merge this PR.

File: cran-comments.md (hunk 1 above)
Comment (fweber144) [reply cran-comments.md:1]: I was also unsure about this, but all the tidyverse packages I just checked (ggplot2, dplyr, and tidyr) have a `cran-comments.md` in their repo and not in `.gitignore`. And when calling `usethis::use_cran_comments()` in a new package with a Git structure, `cran-comments.md` is not added to `.gitignore`. So I think it's ok not to `.gitignore` it.

Comment (paul-buerkner) [reply cran-comments.md:1]: Ok, thank you!

**Comment by fweber144** [comment]
Thanks. I now merged `develop` into `master` (thereby updating `master` from commit dc1c3e47 to commit d831b21c) and will now submit to CRAN.

**Pull request merged**